content/vi/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,90 @@
+---
+reviewers:
+- huynguyennovem
+title: Kubernetes là gì
+content_template: templates/concept
+weight: 10
+card:
+  name: concepts
+  weight: 10
+---
+
+{{% capture overview %}}
+Trang tổng quan của Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+Kubernetes là một nền tảng nguồn mở, khả chuyển, có thể mở rộng để quản lý các ứng dụng được đóng gói và các service, giúp thuận lợi trong việc cấu hình và tự động hoá việc triển khai ứng dụng. Nó là một hệ sinh thái lớn và phát triển nhanh chóng. Các dịch vụ, sự hỗ trợ và công cụ có sẵn rộng rãi.
```

Comment:
I think should change "**Nó** là một hệ sinh thái lớn..." into "**Kubernetes** là một hệ sinh thái lớn...". It is more formal.